docs/appendices/release-notes/5.9.10.rst

```diff
@@ -109,3 +109,9 @@ Fixes
   by the :ref:`sql-restore-snapshot` statement when only a concrete table was
   specified as to be restored. Only the partitioned table definition was falsely
   restored, but not the the actual data.
+
+- Fixed an issue where a filter was not pushed down to the source when the
+  filtered column was aliased as part of a view or sub-select. This could lead
+  to an unnecessary full table scan. e.g.::
+      
```

Comment:
5.9.10 is released, this should be removed. 

I removed 5/9 label because 5.9 is EOL

Reply:
I removed the 5.9 entry.